rele/__main__.py

```diff
@@ -26,11 +26,18 @@ def main():
         "and settings modules. If no settings module is discovered, "
         "defaults will be used.",
     )
-
+    run_parser.add_argument(
+        "--settings",
+        "-s",
+        default=None,
```

Comment:
Isn't the default `settings` in the current path?

Reply:
Surprisingly no. 😄 We do not actually need a settings path anymore. Since #132 was introduced, all other settings are either default or in the case of credentials, taken from the Google environment variables.